eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
I see. However currently `SummaryStatistics` implements `Procedure` that's marked as `Serializable` so it's against the contract.

Should `SummaryStatistics` implement `Consumer` instead? Does it cause any cascading impact (I hope not)?